bots/pr/src/main/java/org/openjdk/skara/bots/pr/CSRCommand.java

```diff
@@ -156,16 +161,10 @@ public void handle(PullRequestBot bot, PullRequest pr, CensusInstance censusInst
             return;
 
         }
-        Issue csr = null;
-        for (var link : jbsIssue.get().links()) {
-            var relationship = link.relationship();
-            if (relationship.isPresent() && relationship.get().equals("csr for")) {
-                csr = link.issue().orElseThrow(
-                        () -> new IllegalStateException("Link with title 'csr for' does not contain issue")
-                );
-            }
-        }
 
+        var csr = jbsIssue.get().links().stream()
```

Comment:
This particular pattern now occurs twice in this file. Perhaps worth moving to a private method at least, to reduce some duplication.

Reply:
Fixed.